### DISCUSSION TITLE
请问npm run build错误，是什么原因

### DISCUSSION THREAD

**Discussion by ZxfBugProgrammer** [spencerwooo/onedrive-vercel-index#164] [Q&A] [answered]
在本地windows环境下运行
npm run dev正常，但执行npm run build报错

具体报错信息如下：

> npm run build
> 
> \> onedrive-vercel-index@0.1.0 build
> \> next build
> 
> info  - Using webpack 5. Reason: Enabled by default https://nextjs.org/docs/messages/webpack5
> info  - Checking validity of types  
> 
> warn - You have enabled the JIT engine which is currently in preview.
> warn - Preview features are not covered by semver, may introduce breaking changes, and can change at any time.
> info  - Creating an optimized production build
> info  - Compiled successfully
> info  - Collecting page data ..(node:3500) ExperimentalWarning: The ESM module loader is experimental.
> info  - Collecting page data ...(node:13556) ExperimentalWarning: The ESM module loader is experimental.
> 
> \> Build error occurred
> Error: Build optimization failed: found pages without a React Component as default export in 
> pages/
> pages/[...path]

**Comment by spencerwooo** [comment, marked as answer]
新版本试一下，不知道之前问题出在哪里了
